test/TestSenseLibrary/app/build.gradle

```diff
@@ -0,0 +1,34 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 21
+    buildToolsVersion "21.1.2"
+
+    defaultConfig {
+        applicationId "nl.sense_os.testsenselibrary"
```

Comment:
@tedschmidt  I am not very familiar with the set up, this one is created by Ronald. But within the Android Studio, there is no other app shown up. Anyway I will try to run the test within the library.